examples/siro_sandbox/sandbox_app.py

```diff
@@ -387,6 +408,9 @@ def sim_update(self, dt):
 
         self._app_state.sim_update(dt, post_sim_update_dict)
 
+        if self._app_state.is_over():
```

Comment:
The old implementation re-runs the tutorial on environment reset: https://github.com/facebookresearch/habitat-lab/blob/69f6fda0f3b67977e5710f9d7917634f0c68300e/examples/siro_sandbox/sandbox_app.py#L324

Do we still have that functionality here?